datasets/hrenwac_para/README.md

```diff
@@ -0,0 +1,145 @@
+---
+annotations_creators:
+- no-annotation
+language_creators:
+- found
+languages:
+- ca
+licenses:
+- cc-by-sa-3.0
+multilinguality:
+- monolingual
+size_categories:
+- n>1M
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
```

Comment:
Isn't this a translation dataset? I've not sure but the paper seems to suggest that http://workshop2013.iwslt.org/downloads/IWSLT-2013-Cettolo.pdf.